crates/frontend/src/utils.rs

```diff
@@ -105,6 +105,7 @@ pub fn get_tenants() -> Vec<String> {
         .unwrap_or(vec![])
 }
 
+#[allow(dead_code)]
```

Comment:
Is the a fix for the clippy warning? Can we let the warning persist and find the root cause?